src/scripts.js

```diff
@@ -8,52 +8,67 @@ const firstName = document.getElementById('firstName');
 const totalStepGoal = document.getElementById('totalStepGoal');
 const waterComsumptionToday = document.getElementById('waterComsumptionToday');
 const weeklyConsumptionList = document.getElementById('weeklyConsumptionList');
+const hoursOfSleep = document.getElementById('hoursOfSleep');
+const qualityOfSleep = document.getElementById('qualityOfSleep');
+const weeklySleepList = document.getElementById('weeklySleepList');
+const allTimeSleepQuality = document.getElementById('allTimeSleepQuality');
+const allTimeSleepHours = document.getElementById('allTimeSleepHours');
 
-let userRepo = new UserRepository(userData);
+// Initialize Data
+const userRepo = new UserRepository(userData);
```

Comment:
I see that you are creating a new UserRepository, but there isn't a userData variable declared/assigned. So what is getting passed through? Maybe adding `const userData = require('../test/datasets/user-sample-data');` will fix the problem?